akka-actor/src/main/scala/akka/actor/ActorSystem.scala

```diff
@@ -467,6 +467,11 @@ abstract class ActorSystem extends ActorRefFactory {
    */
   def eventStream: EventStream
 
+  /**
+   * Java API: Main event bus of this actor system, used for example for logging.
+   */
+  @inline def getEventStream: EventStream = eventStream
```

Comment:
If not sure if adding `@inline` is a good idea.

Reply:
Yeah, skip that.